modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -263,10 +313,8 @@ public Response detail(
     @Tag(name = "Admin", description = "APIs for administrating Xpanse")
     @GetMapping("/health")
     @ResponseStatus(HttpStatus.OK)
-    public SystemStatus health() {
```

Comment:
we used SystemStatus so that we  can add additional fields to it. For now we just have health status.

Reply:
Ok.I will recover it.